docs/recipes/MigratingToI18Next.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Migrating from i18n-js to react-i18next
+description: How to migrate from i18n-js to react-i18next
+tags:
+  - i18n
+last_update:
+  author: Felipe Peña
+publish_date: 2024-09-25
+---
+
+# Migrating from i18n-js to react-i18next
+
+## Overview
+
+In this guide, we will be going through the steps required to migrate your project from `i18n-js` to the `react-i18next` library. The steps are based on the changes outlined in this [PR](https://github.com/infinitered/ignite/pull/2770).
+
+`i18next` will be included in Ignite's version 10. If you're using an earlier version, this guide provides the necessary steps to successfully complete the migration.
+
+## Key steps for migration
+
+### Replace and install dependencies
+
+Replace the `i18n-js` package with the required dependencies for `react-i18next`.
+
+```bash
+yarn remove i18n-js
+yarn add react-i18next i18next
+```
+
+### Update translation initialization
+
+Modify your translation initialization logic to use the `initReactI18next` from `i18next`.
```

Comment:
I'd point the user in the direction of which file it is they should be modifying here (at least in terms of standard previous ignite builds), `app/i18n/i18n.ts`